src/wp-admin/includes/dashboard.php

```diff
@@ -1157,8 +1164,15 @@ function wp_dashboard_rss_output( $widget_id ) {
  * @return bool True on success, false on failure.
  */
 function wp_dashboard_cached_rss_widget( $widget_id, $callback, $check_urls = array(), ...$args ) {
-	$loading    = '<p class="widget-loading hide-if-no-js">' . __( 'Loading&hellip;' ) . '</p><div class="hide-if-js notice notice-error inline"><p>' . __( 'This widget requires JavaScript.' ) . '</p></div>';
 	$doing_ajax = wp_doing_ajax();
```

Comment:
Is there a reason you moved `$doing_ajax` from after `$loading` to before? I can't imagine it making a difference, but checking.

Reply:
It was for no other reason than the variable looked a little orphaned after the `sprintf()`. I figured moving it above at least put its `=` more clearly in line with the one from `$loading`

Reply:
Poor, lonely variable. ;)